### PR TITLE
content updates 2024

### PR DESCRIPTION
**Pull request by onurozer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
...
 
-You can go into most convenience stores (eg. 7-11) and buy a pre-paid simcard
+You can also go into most convenience stores (eg. 7-11) and buy a pre-paid simcard
 from either SingTel or StarHub (or possibly M1).
 
-[SingTel plans](http://info.singtel.com/personal/phones-plans/mobile/singtel-prepaid)
-and
-[StarHub plans](http://www.starhub.com/personal/mobile/prepaid-plans.html).
+- [SingTel plans](https://www.singtel.com/personal/products-services/mobile/prepaid-plans/hi-tourist)
+- [StarHub plans](https://www.starhub.com/personal/mobile/mobile-phones-plans/prepaid-cards/travel-sim.html).
```
Comment (TayKangSheng) [inline README.md:68]: There is an extra `.`

```suggestion
- [StarHub plans](https://www.starhub.com/personal/mobile/mobile-phones-plans/prepaid-cards/travel-sim.html)
```

File: README.md [hunk 2]
```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
...
-## Power (Socket)
+Most hotels have universal plugs that work with Type C (two pin) plugs, but you should consider bringing a travel adapter just in case.
 
-This [bad boy](http://www.iec.ch/worldplugs/typeG.htm) (Type G, BS 1363 UK, 230-240V).
+## Currency
+
+It's a good idea to bring [Singapore dollars
+(SGD)](https://en.wikipedia.org/wiki/Singapore_dollar) when visiting. However
+you can easily get cash out at ATMs after you arrive if necessary. While
+credit cards are widely accepted here, cash is still king, especially at Hawker centers.
```
Comment (TayKangSheng) [inline README.md:83]: I think providing a link to what "hawker centers" are will be helpful. 
https://en.wikipedia.org/wiki/Hawker_centre

File: README.md [hunk 3]
```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
...
 
-Singaporeans often leave tissues on seats in busy food halls (hawker centres
-etc.) while they walk around and buy food. I know you'll be tempted but best
-not to remove the tissue and sit down.
+Singaporeans often leave tissues on tables in busy food courts and hawker centers before they order their food to "reserve" the table. You might be tempted to remove it, but don't. Just find another table and use your own tissue to reserve it.
 
-[Comment: If you need your own tissues, hawker centres are the place to buy
-them. You'll often find an older gentleman or lady offering one, two or more
-packets for a dollar or two.]
+If you don't have a tissue, most hawker centers have an old lady or gentleman offering for a dollar or two, feel free to buy from them.
```
Comment (TayKangSheng) [inline README.md:109]: I think the drink stores of all hawker centers / food courts / coffee shop sells a packet of tissue for roughly $0.30 too.

File: README.md [hunk 4]
```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
...
 - Chewing gum
-  [Comment: Actually you can chew gum like a cow chews cud, but you won't be
-  able to buy any gum in Singapore and when you need to get rid of your gum,
-  make sure you put it in a bin - see "littering" above.]
-- Don't carry [Durian](http://en.wikipedia.org/wiki/Durian) in public transport
-  [Comment: Eating Durian is another must-try experience in Singapore. However, carrying them in public transport is an offence and you will be fined if caught. If you really want to carry it onto public transport, request for them to be packaged on the spot or consider purchasing ones that have already been packed in styrofoam packages wrapped in clingfilm.]
-- Drink and eat in public transport. You can be fined for it. This is especially true for MRT stations and trains, and less so for buses.
-
-[Comment: You *really* don't want to commit even minor crimes here. Singapore
+- Drining and eating in public transport. You can be fined for it. This is especially true for MRT stations and trains.
```
Comment (TayKangSheng) [inline README.md:134]: A missing `k` and also "drinking and eating" reads abit weird to me. Usually its "eating and drinking"? 😅 

```suggestion
- Eating and drinking in public transport. You can be fined for it. This is especially true for MRT stations and trains.
```

File: README.md [hunk 5]
```diff
@@ -257,25 +213,18 @@ A spicy roll and a great option for vegetarians.
 
 This is a very famous Singaporean dish.
 
-Tian Tian in the Maxwell Food Centre is very famous (and I hear half decent).
-If you like to stand in a long queue, please go there at lunch. If you don't go
-there at lunch, they're unlikely to be open.
+Tian Tian in the Maxwell Food Centre is very famous, although the queues can get long, so arrive early if you go there for lunch.
```
Comment (TayKangSheng) [inline README.md:216]: I think we should recommend boon tong kee :) I think the chicken rice is better but less affordable.

File: README.md [hunk 6]
```diff
@@ -304,43 +252,32 @@ Kway Chap is a Teochew dish of flat, broad rice sheets in a soup made with dark
...
 
 ### Putu Piring / Kueh Tutu
 
-Putu Piring is a Traditional Singaporean treat. Steamed rice cake filled with gula melaka (melted palm sugar). Served with grated coconut, squares of pandan leaf, has subtle, sweet vanilla fragrance.
+Putu Piring is steamed rice cake filled with gula melaka (melted palm sugar). Served with grated coconut, squares of pandan leaf, has subtle, sweet vanilla fragrance.
 
-## Indie Bookstore
+## Indie Bookstores
 
 - [BooksActually](https://www.booksactuallyshop.com/pages/about)
```
Comment (TayKangSheng) [inline README.md:276]: Booksactually is closed. this link goes to a gambling site. I think we should remove this section.

**Review by TayKangSheng — APPROVED** [review]
Thanks for updating this doc! Have some recommendations. Otherwise looks good :)

**Comment by onurozer** [comment]
Made all the suggested changes.

**Pull request merged**